server/__init__.py

```diff
@@ -5,14 +5,16 @@
 
 from .rest_slicer_cli import genRESTEndPointsForSlicerCLIsInDocker
 from .handlers import process_annotations
-
+from .constants import *
```

Comment:
It is bad form to `import *`.  It would be clearer to `from . import constants`, and then use `constants.(value)`.  This makes it easier to figure out where functions and values come from when reviewing code.  (In python, explicit is better than implicit.)